db/migrate/20220326155340_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+
```

Comment:
- [ ] Let's add code so it  will create author_id and a corresponding index and  create post_id and a corresponding index. 

```ruby
t.references :author, foreign_key: { to_table: 'users' } 
t.references :post, foreign_key: true
```